.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # default location of `.github/workflows`
+    directory: "/"
+    open-pull-requests-limit: 10
+    schedule:
+      interval: "weekly"
```

Comment:
Why is GH Actions weekly when npm would have more PR's and is daily?

Reply:
Hi @sasial-dev, 
GHA scan is weekly because the actions are not frequently updated. The npm scan is daily because the project's dependencies are updated more often and since those updates could bring significant value to the project (security, performance etc.), the scan is performed more often.